examples/hello-world/step-by-step/higgs/prepare_data.ipynb

```diff
@@ -0,0 +1,363 @@
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!gunzip -c /tmp/nvflare/dataset/input/HIGGS.csv.gz > /tmp/nvflare/dataset/input/higgs.csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "08774782-790c-4a7a-a48c-bbcc11f633cc",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "total 13348436\n",
+      "drwxrwxr-x 2 chester chester       4096 Nov 22 10:05 .\n",
+      "drwxrwxr-x 3 chester chester       4096 Nov 22 09:55 ..\n",
+      "-rw-rw-r-- 1 chester chester 8035497980 Nov 22 10:12 higgs.csv\n",
+      "-rwx------ 1 chester chester 2816407858 May 22  2023 HIGGS.csv.gz\n",
+      "-rw-rw-r-- 1 chester chester 2816865137 Nov 22 09:57 higgs.zip\n"
+     ]
+    }
+   ],
+   "source": [
+    "!ls -al /tmp/nvflare/dataset/input/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cdf4f685-31b3-499b-881e-fc5f73b3f662",
+   "metadata": {},
+   "source": [
+    "### Data Split\n",
+    "\n",
+    "HIGGS dataset contains 11 million instances (rows), each with 28 attributes.\n",
+    "The first 21 features (columns 2-22) are kinematic properties measured by the particle detectors in the accelerator. \n",
+    "The last seven features are functions of the first 21 features; these are high-level features derived by physicists to help discriminate between the two classes. The last 500,000 examples are used as a test set.\n",
+    "\n",
+    "The first column is the class label (1 for signal, 0 for background), followed by the 28 features (21 low-level features then 7 high-level features): lepton  pT, lepton  eta, lepton  phi, missing energy magnitude, missing energy phi, jet 1 pt, jet 1 eta, jet 1 phi, jet 1 b-tag, jet 2 pt, jet 2 eta, jet 2 phi, jet 2 b-tag, jet 3 pt, jet 3 eta, jet 3 phi, jet 3 b-tag, jet 4 pt, jet 4 eta, jet 4 phi, jet 4 b-tag, m_jj, m_jjj, m_lv, m_jlv, m_bb, m_wbb, m_wwbb. For more detailed information about each feature see the original paper.\n",
+    "\n",
+    "Since HIGGS dataset is already randomly recorded, data split will be specified by the continuous index ranges for each client, rather than a vector of random instance indices. We will split the dataset uniformly: all clients has the same amount of data. The output directory \n",
+    "\n",
+    "```\n",
+    "/tmp/nvflare/dataset/output/\n",
+    "\n",
+    "```\n",
+    "\n",
+    "To make it similar to the real world use cases, we put features (CSV file headers) into a file in the input directory.  When we split the file, we make sure each site will has a \"header.csv\" file corresponding to the csv data. In horizontal split. all the header will be the same. but for vertical learning, each site may have different headers. \n",
+    "\n",
+    "\n",
+    "We create a simple python code to split data: called split_csv.py. Let's run this, you will need to wait for few minutes. \n"
```

Comment:
suggest move this line right in front of we actually run "split_csv.py" down there